mobile/src/stateMachine/views/login/welcome.tsx

```diff
@@ -54,6 +54,7 @@ export class WelcomeView extends ViewState {
     async start() {
         Animated.timing(this.state.opacity, {
             toValue: 1, delay: 500, duration: 500,
+            useNativeDriver: true,
```

Comment:
Helps with animation performance. Quote from [https://reactnative.dev/blog/2017/02/14/using-native-driver-for-animated](https://reactnative.dev/blog/2017/02/14/using-native-driver-for-animated) "... we can send everything about the animation to native before it has even started and allows native code to perform the animation on the UI thread without having to go through the bridge on every frame. It is very useful because once the animation has started, the JS thread can be blocked and the animation will still run smoothly."